src/hotspot/share/runtime/synchronizer.hpp

```diff
@@ -114,6 +115,10 @@ class ObjectSynchronizer : AllStatic {
   static void release_monitors_owned_by_thread(TRAPS);
   static void monitors_iterate(MonitorClosure* m);
 
+  static const int NINFLATIONLOCKS = 256;
+  static os::PlatformMutex* gInflationLocks[NINFLATIONLOCKS];
```

Comment:
Why export these?  Why not keep them local to synchronizer.cpp?

Reply:
I was also wondering why.